projectgenerator/gui/generate_project.py

```diff
@@ -348,6 +348,7 @@ def updated_configuration(self):
 
         configuration.epsg = self.epsg
         configuration.inheritance = self.ili2db_options.get_inheritance_type()
+        configuration.tomlfile = self.ili2db_options.get_toml_file()
```

Comment:
Can you remove the `get_` prefix (we also never do that on qgis side). Also remove it on the inheritance type while on it.